src/lib/services/chain-config/api.ts

```
@@ -17,3 +20,12 @@ export const getApiChainConfigs = async (
       },
     })
     .then(({ data }) => parseWithError(zChainConfig.array(), data));
+
+export const getChainProfile = () =>
+  axios.get(CHAIN_PROFILE_URL).then(({ data }) => {
+    const chainProfiles = parseWithError(zChainProfile.array(), data);
+    return chainProfiles.reduce<Record<string, ChainProfile>>(
+      (acc, profile) => ({ ...acc, [profile.pretty_name]: profile }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)